src/data/useCases/postagem/create-postagem.js

```diff
@@ -28,6 +31,10 @@ class CreatePostagemUseCase {
       throw new NotFoundException("id_usuario not found");
     }
 
+    if (!["dce1,dce2,dc3,atletica"].includes(usuario.permissao)) {
+      throw new BadRequestException("User not authorized");
```

Comment:
Tem implementado uma UnauthorizedException, use ela quando for questão de não autorização